icu4j/maven-build/maven-icu4j-datafiles/pom.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0"?>
```

Comment:
Would be nice to have a README.md in this folder, to explain what this does.

People might be familiar with `maven-icu4j`, `maven-icu4j-charset`, `maven-icu4j-localespi` because they match the public artifacts that they (probably) used for years.
But no clue what this is / does.

Reply:
Copied over Readmes from the other submodules.  The Readmes are temporary. 

We can fill out more documentation if things become more permanent, which would require multiple steps first (landing this Maven build, replacing the Ant build with the Maven build, etc.)